content/work/timedesigner.md

```diff
@@ -7,13 +7,49 @@ date: 2018-07-15
 image:
   src: projects/tegy/grid-desktop.jpg
   alt: School schedule grid
+fullgrid:
+  - img: work/timedesigner/full-grid.jpg
+    alt: a completed scenario with experiences laid out in a grid
+experiencegrid:
+  - img: work/timedesigner/experience-editing.jpg
+    alt: browser inspector showing grid markup and css for a single experience
+emptygridoverlay:
+  - img: work/timedesigner/empty-grid-overlay.jpg
+    alt: Firefox Grid Inspector tool including an overlay of the grid lines
+scenariogrid:
+  - img: work/timedesigner/setting-up-scenario-grid.jpg
+    alt: an empty resource planning grid showing the start and end times
+overlapexperience:
+  - img: work/timedesigner/overlap-experience.jpg
+    alt: a row with overlapping experiences
 people:
   - &furman
     name: Furman Brown
-    # face: furman-brown.jpg
+    face: furman-brown.jpg
     url: https://www.tegy.tools/profile/furman/profile
     title: Founder
     venue: *client
+tags:
+  - _post
+  - Case Study
+sample:
+  desktop:
+    src: tegy/timedesigner-desktop.jpg
+    alt: Chicago K-8 School schedule iteration
+  mobile:
+    src: tegy/mobile-summary.jpg
+    alt: Chicago K-8 School schedule iteration
```

Comment:
I'm not convinced that this works. We show the "mobile view" in two places, and in both it's very clear that this app wasn't actually designed to work on small screens. I don't like the idea of forcing it onto a phone just to be the same as other case-studies. While consistent design is great - I think one of our selling points is that we do custom bespoke work, based on the needs of the client. Why wouldn't we also adapt our case-studies to match those custom needs?